sql/mozfun/glam/histogram_generate_functional_buckets/metadata.yaml

```diff
@@ -6,7 +6,7 @@ description: |
   A functional bucketing algorithm. The bucket index of a given sample is
   determined with the following function:
 
-  i = ⌊n log<sub>base</sub>(𝑥)⌋
+  i = \floor{n log<sub>base</sub>(x)}
```

Comment:
I believe this is the LaTeX way to write this, but am also open to alternatives

Reply:
Maybe something like:

```
$$ \lfloor{n log_{\text{base}}{(x)}}\rfloor $$
```